src/vtt-segment-loader.js

```diff
@@ -307,19 +293,16 @@ export default class VTTSegmentLoader extends SegmentLoader {
    * @private
    */
   parseVTTCues_(segmentInfo) {
-    let decoder;
-    let decodeBytesToString = false;
-
-    if (typeof window.TextDecoder === 'function') {
-      decoder = new window.TextDecoder('utf8');
-    } else {
-      decoder = window.WebVTT.StringDecoder();
-      decodeBytesToString = true;
-    }
-
+    // It's not necessary to decode text from VTT files so
+    // we'll pass in a fake decoder
+    const fakeDecoder = {
+      decode(d) {
+        return d;
+      }
+    };
     const parser = new window.WebVTT.Parser(window,
                                             window.vttjs,
-                                            decoder);
+                                            fakeDecoder);
```

Comment:
so, should we just be using the StringDecoder? Don't think it makes a huge difference, though.

Reply:
I'll use that instead since the original issue seems to stem from transforming the array buffer to text.